<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1384</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1581</width>
      <height>761</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Данные</string>
     </attribute>
     <widget class="QTabWidget" name="tablesViewWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1571</width>
        <height>711</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>НИР</string>
       </attribute>
       <widget class="QComboBox" name="comboBoxSort">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>650</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Без сортировки</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сортировка по каждому столбцу</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сортировка по Увеличению Кода</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сортировка по Убыванию Кода</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="changeButton">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>650</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="appendButton">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>650</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeButton">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>650</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="filterButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>650</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Фильтровать</string>
        </property>
       </widget>
       <widget class="QPushButton" name="resetFiltersButton">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>650</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Сбросить фильтры</string>
        </property>
       </widget>
       <widget class="QLabel" name="NirLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="NirViewWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>1561</width>
          <height>621</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Финансирование</string>
       </attribute>
       <widget class="QLabel" name="FinancialLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>351</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="FinancialViewWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>651</width>
          <height>621</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>ГРНТИ</string>
       </attribute>
       <widget class="QLabel" name="GrntiLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="GrntiViewWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>701</width>
          <height>621</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>ВУЗы</string>
       </attribute>
       <widget class="QLabel" name="VuzLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>301</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="VuzViewWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>1561</width>
          <height>621</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Анализ</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Финансирование</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выход из программы</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1384</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
